liana-gui/src/app/view/receive.rs

```diff
@@ -348,6 +398,7 @@ pub fn verify_address_modal<'a>(
 }
 
 pub fn qr_modal<'a>(qr: &'a qr_code::Data, address: &'a String) -> Element<'a, Message> {
+    let max_width = if address.len() > 64 { 600 } else { 400 };
```

Comment:
I think we are re-using this modal for the bip21. In which case we should rename the address to param to something more general